### PR TITLE
Folder icon issue

### PR DESCRIPTION
**Pull request by jamran7**
This is the 80% fix. Still need to figure out what will determine if a file is a directory.
Adobe repository: delete this if you are seeing this as a pull request. This is supposed to merge into an independent project.

### CODE REVIEW

File: src/htmlContent/about-dialog.html [hunk 1]
```diff
@@ -14,6 +14,8 @@ <h1 class="dialog-title">{{Strings.ABOUT}}</h1>
                 </p>
 
                 <p class="dialog-message"><!-- $NON-NLS$ -->Copyright 2012 - present Adobe Systems Incorporated and its licensors. All rights reserved.</p>
+                
+                <p class="dialog-message"><b>This edition has been edited by: Chris, Boyce, James and Logan</b></p>
```
Comment (zaggino) [inline src/htmlContent/about-dialog.html]: 😮

**Review by zaggino — CHANGES_REQUESTED** [review]
those readme and "this edition has been edited by" changes don't belong to a pull request

**Comment by petetnt** [comment]
Hi @jamran7,

pretty sure this was already fixed in #12807, correct?

**Comment by ficristo** [comment]
@jamran7 if you use a fork please never point to `"healthDataServerURL": "https://health.brackets.io/healthDataLog"` which is the production url for the health report of Brackets.
Block it or change to the dev url instead.
Per first comment closing as this should not be merged here.

**Comment by ficristo** [comment]
@jamran7 if you have any question, you are more than welcome to ask them here on github or on slack.
It seems to me you are a group of students that are trying to fix some issues, I don't know if you can, but we are open to discuss the possible solutions.

**Comment by petetnt** [comment]
Here's additional contact information:

Slack: [Brackets on Slack](https://brackets.slack.com/) (You can join by sending a mail to admin@brackets.io with the subject line slack registration request specifying the email addresses you would like to register).
Developers mailing list: http://groups.google.com/group/brackets-dev

**Pull request closed without merging**